plugins/modules/azure_rm_galleryimageversion_info.py

```diff
@@ -230,20 +232,37 @@ def listbygalleryimage(self):
         self.url = self.url.replace('{{ image_name }}', self.gallery_image_name)
 
         try:
-            response = self.mgmt_client.query(self.url,
-                                              'GET',
-                                              self.query_parameters,
-                                              self.header_parameters,
-                                              None,
-                                              self.status_code,
-                                              600,
-                                              30)
-            results = json.loads(response.text)
+            skiptoken = None
+
+            while True:
+                if skiptoken:
+                    self.query_parameters['skiptoken'] = skiptoken
+
+                response = self.mgmt_client.query(self.url,
+                                                'GET',
```

Comment:
line 342, 243, 244, 245, 246's continuation line under-indented for visual indent